blocks/layout-grid/src/grid/variations.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/** @typedef {import('@wordpress/blocks').WPBlockVariation} WPBlockVariation */
+
+/**
+ * Internal dependencies
+ */
+import ColumnIcon from './../icons';
+/**
+ * Template option choices for predefined columns layouts.
+ *
+ * @type {WPBlockVariation[]}
+ */
+const variations = [
+	{
+		name: 'one-column',
+		title: __( 'One' ),
```

Comment:
I'm working on adding i18n support for the Layout Grid block and I bumped into a case where these strings `One`, `Two`, `Three`, and `Four` are being missed because they don't reference the `layout-grid` domain as the rest of strings.

Not sure if it's an issue because I couldn't find the translations on either of both GlotPress projects ([Gutenberg](https://translate.wordpress.org/projects/wp-plugins/gutenberg/stable/es/default/?filters%5Bterm%5D=Four&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc) / [Layout Grid](https://translate.wordpress.org/projects/wp-plugins/layout-grid/stable/es/default/?filters%5Bterm%5D=Four&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)) or it's expected to use the default domain, @enejb I'd appreciate it if you could provide further insights regarding this, thanks!

Reply:
I think for one reason or another I thought that in Gutenberg we had the values translated. So I left them as they were taking the value from core. Since we didn’t translate blocks that were not living outside of Gutenberg. (.
I used the columns block as an example for this code. )  Or it might have been a typo and I missed adding the domain part of the function.

Reply:
Great, thank you very much for info 🙇 !

> I think for one reason or another I thought that in Gutenberg we had the values translated. So I left them as they were taking the value from core.

Yep, I was also surprised that number strings like "One" / "Two" aren't translated in Gutenberg, however, I noticed that we actually have them but with a dot at the end (i.e. "One.") 😅 .

> Since we didn’t translate blocks that were not living outside of Gutenberg. (.

This is part of the work I'm doing for the Translation Pipeline Improvement project, although for the strings that are used only in the native version of the editor, they are currently being translated as part of the WordPress app.

> I used the columns block as an example for this code. ) Or it might have been a typo and I missed adding the domain part of the function.

Ok, it's still unclear to me whether to use a domain or not within plugins, per [this documentation](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/) about plugin translations, I understand that it's recommended to use a domain but I guess it's ok to fall back to default (Gutenberg in this case)  if we know that the string is present there.

In any case, with the work I'm doing for the pipeline improvements, these strings will be added to the localization strings files and translated as strings of the WordPress app ([example reference](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4411/files#diff-d116ecb4f40d4fa795c1a136e7fddfef1850a015725f83835d7d9ee33f04ee64)).